### PR TITLE
Improve the readme file to provide clear instructions on how to build

### PR DESCRIPTION
**Pull request by SergioEstevao**
Updates the readme to provide clear instructions on how to build instead of redirecting to other sites.

Needs review: @astralbodies

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,61 +1,91 @@
...
+`pod install`
 
-GitHub: https://github.com/wordpress-mobile/WordPress-iOS/
+#### SwiftLint
 
-### How to Contribute
+We use [SwiftLint](https://github.com/realm/SwiftLint) to enforce a common style for Swift code. Xcode will show a warning if you don't have it installed. The app should build and work without it, but if you plan to write code, you are encouraged to install it. No commit should have lint warnings or errors.
 
-http://make.wordpress.org/mobile/handbook/pathways/ios/how-to-contribute/
+SwiftLint runs automatically when you build the project, but you can run it manually from the command line with `rake lint`
```
Comment (astralbodies) [inline README.md]: I'd put the command on a new line like you did in the third-party tools section.

File: README.md [hunk 2]
```diff
@@ -1,61 +1,91 @@
...
 
-## Building
+If your code has any style violations, you can try to automatically correct them by running `rake lint:autocorrect`. Otherwise you have to fix them manually.
```
Comment (astralbodies) [inline README.md]: Same as my last comment - new line before the command

File: README.md [hunk 3]
```diff
@@ -1,61 +1,91 @@
...
 
-### CocoaPods
+Launch the workspace by running `rake xcode` from the command line. This will ensure any dependencies are ready before launching Xcode. 
+You can also open the project by double clicking on WordPress.xcworkspace file, or launching Xcode and choose File > Open and browse to WordPress.xcworkspace.
 
-WordPress for iOS uses CocoaPods (http://cocoapods.org/) to manage third party libraries.  Trying to build the project by itself (WordPress.xcproj) after launching will result in an error, as the resources managed by CocoaPods are not included.
+### Setup Credentials
 
-Run `rake dependencies` from the command line to install dependencies for the project. This will ensure the current supported version of CocoaPods is used. If you have the same version installed in your system, you can also run `pod install`.
+In order to login on WordPress.com using the app you will need to create an account over at [WordPress.com Developer Portal](https://developer.wordpress.com).
```
Comment (astralbodies) [inline README.md]: s/login on/login to

Comment (astralbodies) [inline README.md]: s/over at/over at the

File: README.md [hunk 4]
```diff
@@ -1,61 +1,91 @@
...
 
-More information on building the project and the tools required can be found in [the handbook](https://make.wordpress.org/mobile/handbook/pathways/ios/tools-requirements/).
+You can read more about [Code Style Guidelines](https://github.com/wordpress-mobile/WordPress-iOS/wiki/WordPress-for-iOS-Style-Guide) we adopted, and
+how we're organizing branches in our repository in the [Contribution Guide](https://make.wordpress.org/mobile/handbook/pathways/ios/how-to-contribute/).
 
-## Logging in
+## Need help to build or hack? ##
 
-In order to login to a WordPress.com account, you will need to create an account over at https://developer.wordpress.com. The only account you will be able to login in with is the one affiliated with your developer account. Once you have an account and a corresponding app id and app secret, you will need to setup a ~/.wpcom_app_credentials file as detailed [here](http://make.wordpress.org/mobile/handbook/pathways/ios/tutorials-guides/#3-%c2%a0setup-wpcom_app_credentials). For more details see http://developer.wordpress.com/2014/07/04/authentication-improvements-for-testing-your-apps/.
+Say hello on our [Slack][4] channel: `#mobile`.
```
Comment (astralbodies) [inline README.md]: I think this is supposed to be a link to our Slack instance?

**Comment by SergioEstevao** [comment]
@astralbodies ready for another look.

File: README.md [hunk 5]
```diff
@@ -64,7 +75,7 @@ how we're organizing branches in our repository in the [Contribution Guide](http
 
 ## Need help to build or hack? ##
 
-Say hello on our [Slack][4] channel: `#mobile`.
+Say hello on our [Slack](wordpress.slack.com) channel: `#mobile`.
```
Comment (astralbodies) [inline README.md]: Oops this needs an https:// in front of it - but we should probably reference https://chat.wordpress.org for the instructions on how to join.

**Comment by SergioEstevao** [comment]
@astralbodies ready again for another look.

**Comment by astralbodies** [comment]
:shipit:

**Pull request merged**